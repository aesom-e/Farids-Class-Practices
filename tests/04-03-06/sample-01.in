I met Alice and Bob at the Park on Sunday