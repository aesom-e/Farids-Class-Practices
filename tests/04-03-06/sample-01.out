['I', 'Alice', 'Bob', 'Park', 'Sunday']